packages/lesswrong/server/type3.ts

```diff
@@ -37,51 +39,76 @@ const type3ApiRequest = async (
   }
 }
 
-const postWithAudioProjection = {_id: 1, slug: 1} as const;
+const isDocumentAllowedType3Audio = (document: DbPost | DbTag, collectionName: 'Posts' | 'Tags') => {
+  if (collectionName === 'Posts') {
+    return isPostAllowedType3Audio(document as DbPost);
+  } else {
+    return isTagAllowedType3Audio(document as DbTag);
+  }
+} 
+
+const documentWithAudioProjection = {_id: 1, slug: 1} as const;
 
-type PostWithAudio = Pick<DbPost, keyof typeof postWithAudioProjection>;
+type PostWithAudio = Pick<DbPost, keyof typeof documentWithAudioProjection>;
+type TagWithAudio = Pick<DbTag, keyof typeof documentWithAudioProjection>;
```

Comment:
I think these are the same type (since both `_id` and `slug` are just `string` for both `DbPost` and `DbTag`; they don't have different nullability in one case from the other).